src/components/editMeasure/testCases/components/editTestCase/groupCoverage/groupCoverageNav/GroupCoverageNav.tsx

```diff
@@ -35,14 +34,8 @@ const GroupCoverageNav = ({
   includeRAV,
   model,
 }: Props) => {
-  const featureFlags = useFeatureFlags();
-
   // TODO Remove parameter when either of these flags are removed, or when both feature flags are removed
-  const showRAVTab =
-    includeRAV &&
-    (model === "qiCore"
-      ? featureFlags?.QICoreIncludeRAVValues
-      : featureFlags?.QDMIncludeRAVValues);
+  const showRAVTab = includeRAV;
```

Comment:
I agree with the TODO. Please remove this const and use `includeRAV` where needed.